### PR TITLE
BETA: Register mwa.is-a.dev

### PR DESCRIPTION
**Pull request by Totallynotmwa**
Added `mwa.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

**Comment by Totallynotmwa** [comment]
Figuring out how to make a redirect bro☠️

File: domains/totallynotmwa.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Totallynotmwa",
+        "email": "totallynot@mwa.lol"
+    },
+    "record": {
+        "A": ["192.0.2.1"]
```
Comment (CuteDog5695) [inline domains/totallynotmwa.json:7]: put an actual IP, not a local IP

File: domains/mwanothere.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Totallynotmwa",
+        "email": "totallynot@mwa.lol"
+    },
+    "record": {
+        "A": ["192.0.2.1"]
```
Comment (CuteDog5695) [inline domains/mwanothere.json:7]: put an actual IP, not a local IP

File: domains/mwa.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Totallynotmwa",
+        "email": "totallynot@mwa.lol"
+    },
+    "record": {
+        "A": ["192.0.2.1"]
```
Comment (CuteDog5695) [inline domains/mwa.json:7]: put an actual IP, not a local IP

**Review by iostpa — CHANGES_REQUESTED** [review]

File: domains/totallynotmwa.json (hunk 1 above)
Comment (Totallynotmwa) [reply domains/totallynotmwa.json]: There i changed it all

File: domains/totallynotmwa.json [hunk 4]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Totallynotmwa",
+        "email": "totallynot@mwa.lol"
+    },
+    "record": {
+        "URL": "mwa.lol"
```
Comment (CuteDog5695) [inline domains/totallynotmwa.json:7]: doesnt have protocol

File: domains/mwanothere.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Totallynotmwa",
+        "email": "totallynot@mwa.lol"
+    },
+    "record": {
+        "URL": "mwa.lol"
```
Comment (CuteDog5695) [inline domains/mwanothere.json:7]: doesnt have protocol

File: domains/mwa.json [hunk 6]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Totallynotmwa",
+        "email": "totallynot@mwa.lol"
+    },
+    "record": {
+        "URL": "mwa.lol"
```
Comment (CuteDog5695) [inline domains/mwa.json:7]: doesnt have protocol

**Review by iostpa — CHANGES_REQUESTED** [review]

**Review by iostpa — APPROVED** [review]
seems fine to me now

**Pull request merged**